package.json

```diff
@@ -41,9 +41,7 @@
     "spellcheck": "mdspell -r --en-gb --ignore-acronyms --ignore-numbers --no-suggestions '*.md' '**/*.md' '!**/node_modules/**/*.md' '!backpack/**/*.md' '!backpack-react-native/**/*.md' '!backpack-ios/**/*.md'  '!backpack-android/**/*.md'",
     "spellcheck:interactive": "mdspell --en-gb --ignore-acronyms --no-suggestions --ignore-numbers '*.md' '**/*.md' '!**/node_modules/**/*.md' '!backpack/**/*.md' '!backpack-react-native/**/*.md' '!backpack-ios/**/*.md'  '!backpack-android/**/*.md'",
     "update-bpk-dependencies": "npx npm-check-updates -n '/^bpk\\-(.*)$/' -u",
-    "lighthouse": "babel-node scripts/lighthouse/lighthouse.js",
-    "snyk-protect": "snyk protect",
-    "prepublish": "npm run snyk-protect"
```

Comment:
Did you mean to remove these?

Reply:
Ah yeah will add them back they were just taking a long time to run once post install took place